appveyor.yml

```diff
@@ -13,7 +13,7 @@ environment:
 configuration:
   - Debug
   - Release
-version: 3.6.2.{build}
+version: 3.7.0.{build}
```

Comment:
@Sand3r- has already bumped version number to this one. You need to increment it even more & update w.r.t. `develop`.